internal/featuresupport/support_levels_list.go

```diff
@@ -141,6 +157,10 @@ var SupportLevelsList = models.FeatureSupportLevels{
 				FeatureID:    models.FeatureSupportLevelFeaturesItems0FeatureIDSINGLENODEEXPANSION,
 				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelSupported,
 			},
+			{
+				FeatureID:    models.FeatureSupportLevelFeaturesItems0FeatureIDLVM,
+				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelSupported,
```

Comment:
this should be FeatureSupportLevelFeaturesItems0SupportLevelDevPreview and please add a new one for 4.12 (copy 4.11 and change it to supported in 4.12)